src/clients/omsApiProxy.ts

```diff
@@ -0,0 +1,50 @@
+import { AppClient, InstanceOptions, IOContext } from '@vtex/api'
+
+/**
+ * Used to perform calls on OMS API in the VTEX infrastructure, to which you must declare an outbound policy.
+ *
+ * Policy:
+ * {
+ *   "name": "vtex.oms-api-proxy:oms-order-information"
+ * },
+ *
+ */
+export class OMSProxy extends AppClient {
+  constructor(context: IOContext, options?: InstanceOptions) {
+    super('vtex.oms-api-proxy@0.x', context, options)
+  }
+
+  public orders(id: string): Promise<string> {
+    return this.http.get(`/orders/${id}`, {
```

Comment:
```suggestion
    return this.http.get<Order>(`/orders/${id}`, {
```

Don't we have the returning types for all of these methods?